### PR TITLE
Add Bitcoin-XT v0.11E

### PR DESCRIPTION
**Pull request by bibnumber**
Casks works audite etc. as described. Tried to implement the #{version} string in url but it broke the cask, maybe because of the capital letter E at the end of version?

Apart from that: the suggested method for [shasum the appcast with curl](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md) did not work for me (zsh nor sh) as it returns empty string. That ought to be updated for that i spending most of the time figuring out why the sha check on the file didn't audit correctly until i figured **that the appcast curl snippet just returned bogus sha**.
Best regards

```
# Committer: Steven <SKN@localhost.local>
#
# On branch bitcoin-xt
# Changes to be committed:
#   new file:   Casks/bitcoin-xt.rb
#
# Untracked files:
#   BitcoinXT-0.11E.dmg
#
#
# It took 2.34 seconds to enumerate untracked files. 'status -uno'
# may speed it up, but you have to be careful not to forget to add
# new files yourself (see 'git help status').
# ------------------------ >8 ------------------------
# Do not touch the line above.
# Everything below will be removed.
diff --git a/Casks/bitcoin-xt.rb b/Casks/bitcoin-xt.rb
new file mode 100644
index 0000000..c9c5e60
--- /dev/null
+++ b/Casks/bitcoin-xt.rb
@@ -0,0 +1,20 @@
+cask 'bitcoin-xt' do
+  version '0.11E'
+  sha256 '1a1dcfcaef0d3cf87b8994eca25aa53789650220c783488e10eb74eac287d40a'
+
+  url 'https://github.com/bitcoinxt/bitcoinxt/releases/download/v0.11E/BitcoinXT-0.11E.dmg',
+      referer: 'https://github.com/bitcoinxt/bitcoinxt/releases/',
+      cookies: {
+                 'i_follow_redirects' => 'yes',
+                 'logged_in'          => 'no',
+               }
+  appcast 'https://github.com/bitcoinxt/bitcoinxt/releases.atom',
+          checkpoint: '843d6ef3717eaf20b2e88f78e3eacae10dc745a0c134f34187acfdbfcfdb4d8d'
+  # Vendor dev and release is on github official homepage is hosted separatly
+  name 'Bitcoin XT',
+       'Bitcoin-XT'
+  homepage 'https://bitcoinxt.software/'
+  license :oss
+
+  app 'Bitcoin-XT.app'
+end
```

### CODE REVIEW

File: Casks/bitcoin-xt.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+cask 'bitcoin-xt' do
+  version '0.11E'
+  sha256 '1a1dcfcaef0d3cf87b8994eca25aa53789650220c783488e10eb74eac287d40a'
+
+  url 'https://github.com/bitcoinxt/bitcoinxt/releases/download/v0.11E/BitcoinXT-0.11E.dmg',
+      referer: 'https://github.com/bitcoinxt/bitcoinxt/releases/',
```
Comment (vitorgalvao) [inline Casks/bitcoin-xt.rb]: Are you sure this is needed?

File: Casks/bitcoin-xt.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+cask 'bitcoin-xt' do
+  version '0.11E'
+  sha256 '1a1dcfcaef0d3cf87b8994eca25aa53789650220c783488e10eb74eac287d40a'
+
+  url 'https://github.com/bitcoinxt/bitcoinxt/releases/download/v0.11E/BitcoinXT-0.11E.dmg',
```
Comment (vitorgalvao) [inline Casks/bitcoin-xt.rb]: `url "https://github.com/bitcoinxt/bitcoinxt/releases/download/v#{version}/BitcoinXT-#{version}.dmg"` will work. Perhaps you forgot to change quotes do double quotes?

File: Casks/bitcoin-xt.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      referer: 'https://github.com/bitcoinxt/bitcoinxt/releases/',
+      cookies: {
+                 'i_follow_redirects' => 'yes',
+                 'logged_in'          => 'no',
+               }
+  appcast 'https://github.com/bitcoinxt/bitcoinxt/releases.atom',
+          checkpoint: '843d6ef3717eaf20b2e88f78e3eacae10dc745a0c134f34187acfdbfcfdb4d8d'
+  # Vendor dev and release is on github official homepage is hosted separatly
+  name 'Bitcoin XT',
```
Comment (vitorgalvao) [inline Casks/bitcoin-xt.rb]: Leave just first `name`.

File: Casks/bitcoin-xt.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+  url 'https://github.com/bitcoinxt/bitcoinxt/releases/download/v0.11E/BitcoinXT-0.11E.dmg',
+      referer: 'https://github.com/bitcoinxt/bitcoinxt/releases/',
+      cookies: {
+                 'i_follow_redirects' => 'yes',
+                 'logged_in'          => 'no',
+               }
+  appcast 'https://github.com/bitcoinxt/bitcoinxt/releases.atom',
+          checkpoint: '843d6ef3717eaf20b2e88f78e3eacae10dc745a0c134f34187acfdbfcfdb4d8d'
+  # Vendor dev and release is on github official homepage is hosted separatly
```
Comment (vitorgalvao) [inline Casks/bitcoin-xt.rb]: Remove comment.

File: Casks/bitcoin-xt.rb (hunk 2 above)
Comment (vitorgalvao) [inline Casks/bitcoin-xt.rb]: Needs a `# github.com/bitcoinxt/bitcoinxt was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by bibnumber** [comment]
All right thanks for the input. I'll fix it up over the weekend.

**Comment by vitorgalvao** [comment]
Thank you. I’ll just squash up your commits and merge.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 4024927b43bd276d35cf4dc9d167281bb72e7958. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**